### ISSUE TITLE
Confusing instructions from JuliaRegistrator

### ISSUE THREAD

**Issue by cscherrer** [JuliaRegistries/Registrator.jl#319]
Hi, I found this warning message really confusing:
> Also, note the warning: Version 0.1.3 skips over 0.1.2
This can be safely ignored. However, if you want to fix this you can do so. Call register() again after making the fix. This will update the Pull request.

There's some discussion on it here:
https://discourse.julialang.org/t/confusing-instructions-from-juliaregistrator/53833

Thanks @ericphanson :)

**Comment by ericphanson** [comment]
No, I think it’s a RegistryCI issue, since the confusing wording is in the automerge message

**Comment by DilumAluthge** [comment]
I believe the wording in question is in the Registrator comment, right?

@cscherrer Is this what you're referring to? https://github.com/cscherrer/NestedTuples.jl/commit/cf2fcc29436b5e5411e8db3c6b5aa858555b18b4#commitcomment-46276353

**Comment by cscherrer** [comment]
@DilumAluthge yep that's it

**Comment by DilumAluthge** [comment]
Now that we have AutoMerge, maybe we should just remove this message from Registrator.

**Comment by ericphanson** [comment]
Whoops, I mixed it up with the automerge comment. Agreed that it’s better for these to be just in one place, ie in the automerge message(except for errors where it’s actually not going to open the PR or something like that).

**Comment by DilumAluthge** [comment]
> Whoops, I mixed it up with the automerge comment. Agreed that it’s better for these to be just in one place, ie in the automerge message(except for errors where it’s actually not going to open the PR or something like that).

I agree. All warnings and non-fatal errors should go in the AutoMerge comment.
